src/library/trackcollection.cpp

```diff
@@ -510,4 +514,80 @@ StorageUnicodeFunctions::likeFunction(sqlite3_context *p,
                                     itrString, endString, E));
 }
 */
+//Private function
+
+bool TrackCollection::settingsPathStatus(){
+    QFileInfo fle(m_pConfig->getSettingsPath());
+    QFile::Permissions permit = fle.permissions();
+    permit = permit & MASK;
+    const short int NONE = QFile::NoOptions;
```

Comment:
but this is the cause of the error, I am using these words instead of just constant literals, which is giving the erro for the windows, should I use if else structure otherwise